articles/the-role-of-artificial-intelligence-for-smarter-cybersecurity/index.md

```diff
@@ -0,0 +1,53 @@
+
+### Network Security
+
+Network security mainly focuses on the understanding of network topography and creating a security policy of an organization. Securities policies can help to distinguish legitimate network connections from illegitimate ones. But drafting and maintaining policies for multiple networks may be challenging.
+
+Often, some organizations may not have a precise naming convention for workloads and applications. This means that network security teams have to spend significant time attaching different workloads to given applications.
+
+These activities are time-consuming, and the use of artificial intelligence technology can come in handy. AI can expedite these processes by learning and observing traffic patterns and suggesting security policies. This way, organizations save on time, resources, and effort. The organization can allocate these to other areas of technological advancement or development.
+
+### Data Centers
+
+Cyber attackers target [data centers](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/what-is-a-data-center.html) in an attempt to obtain critical data. Cybersecurity has improved over the years with advancements in technology. Being aware of this, cybercriminals regularly develop more advanced malware strains to ensure successful cyber-attacks on organizations&#39; networks.
+
+Organizations can deploy artificial intelligence technology in their data centers to enhance data security. To ensure data security, AI essentially learns normal network behavior. It then detects cyber threats with regard to the variations from the regular network behavior.
+
+The application of AI in a data center can help identify security loopholes and detect malware in data center systems. Most importantly, AI-based cybersecurity can analyze or screen outgoing and incoming data security threats.
+
+### Threat Hunting
+
+The asymmetric nature of cybersecurity is an inherent problem that seems to empower the acts of attackers. With this state, cyber-attackers need to only be successful once by exploiting just a single weakness while the defenders must be successful every time. Preventing threats requires more than just ensuring security data. It calls for combing threats on an entire [information technology stack](https://www.pcmag.com/encyclopedia/term/technology-stack).
+
+In this case, AI is essential to identify abnormalities, outliers, and patterns in all data without the need for applying fixed rules. Human analysts depend on this output for investigation and action. This is what is often referred to as threat hunting in security language. It involves narrowing down threats by collaborating security analytics and machine intelligence, and advanced human cognition.
```

Comment:
Again saying AI is performing threat hunting and passing it down to the security team is on the surface-level, What we want to provide the reader with is value. How is AI performing threat hunting? Give an intuitive example, if possible. I am not asking you to describe the algorithm in detail. But what algorithms are we talking about and how do they fit in the overall problem of threat hunting.